scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/task/internal/InternalTask.java

```diff
@@ -1033,6 +1033,7 @@ protected TaskLauncherInitializer getDefaultTaskLauncherInitializer(InternalJob
         TaskLauncherInitializer tli = new TaskLauncherInitializer();
         tli.setTaskId(getId());
         tli.setJobOwner(job.getJobInfo().getJobOwner());
+        tli.setRestUrl(System.getProperty("rest.server.url"));
```

Comment:
Extract in a constant that is publicly available and prefix it with _pa_ or _proactive_ to help debugging?

It would be also convenient to standardize naming among parameters, properties, etc. which have been added for passing the Scheduler REST URL.